src/logic/prepare/provider/security.ts

```diff
@@ -0,0 +1,96 @@
+import type {
+  ApiKeyPlacement,
+  ApiKeySecurityScheme,
+  SecurityScheme,
+  SecurityValues,
+} from '@superfaceai/ast';
+import {
+  HttpScheme,
+  prepareSecurityValues,
+  SecurityType,
+} from '@superfaceai/ast';
+import inquirer from 'inquirer';
+
+export async function selectSecurity(
+  provider: string
+): Promise<{ value?: SecurityValues; scheme?: SecurityScheme }> {
+  const scheme = await enterSecuritySchema(provider);
+
+  if (scheme === 'none') {
+    return {};
+  }
+
+  return {
+    value: prepareSecurityValues(provider, [scheme])[0],
+    scheme,
+  };
+}
+
+async function enterSecuritySchema(
+  provider: string
+): Promise<SecurityScheme | 'none'> {
+  const schema: 'api key token' | 'bearer token' | 'basic' | 'digest' | 'none' =
+    (
+      await inquirer.prompt<{
+        schema: 'api key token' | 'bearer token' | 'basic' | 'digest' | 'none';
+      }>({
+        name: 'schema',
+        message: `Select a security schema for ${provider}:`,
+        type: 'list',
+        choices: ['api key token', 'bearer token', 'basic', 'digest', 'none'],
```

Comment:
Doesn't this sound/look better?

```suggestion
        choices: ['API Key', 'Bearer Token', 'Basic Auth', 'Digest', 'none'],
```